dsm_arnitel2045_175.xml.fdm_material

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generic ABS profile. The data in this file may not be correct for your specific machine.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+    <metadata>
+        <name>
+            <brand>DSM</brand>
+            <material>Elastomer</material>
```

Comment:
Our intent was to make it so specific that one set of profiles could print for it. If there is some other elastomer, such as TPU, then it might require different temperatures and you'd need the user to select the correct profile instead of Cura filtering it for him.